R/impact_central.R

```diff
@@ -431,24 +431,22 @@ impact_by_year_of_vaccination_activity_type <- function(
   baseline_burden, focal_burden, fvps, vaccination_years) {
```

Comment:
The docs for this say "this method produces multiple, activity-specific impact ratios" this is no longer true, we should update this. It only work for 1 activity type.

Reply:
More generally, the idea of these functions was that they would match the format from the paper. After these changes are we still satisfied that they match what the paper describes? @kgaythorpe